SU-SWS/sulgryphon-nextjs
# Refactored to use GraphQL API

# READY FOR REVIEW, 
DO NOT MERGE until the Drupal application gets updated.

# Summary
- Replace Jsonapi with Graphql data fetching.

# Review By (Date)
- No deadline


# Steps to Test
1. using the latest version of the Drupal stack checkout this branch
2. `yarn`
3. create some content and verify it displays on the FE

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
